packages/functional-tests/src/functional-tests.local.spec.ts

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { CosmosClient } from '@azure/cosmos';
+import { CosmosClientWrapper, CosmosContainerClient } from 'azure-services';
+import { GuidGenerator, HashGenerator, ServiceConfiguration, System } from 'common';
+import { isEmpty } from 'lodash';
+import { ConsoleLoggerClient, GlobalLogger } from 'logger';
+import {
+    OnDemandPageScanRunResultProvider,
+    PartitionKeyFactory,
+    RunState,
+    ScanResultResponse,
+    ScanRunResultResponse,
+} from 'service-library';
+import { A11yServiceClient, A11yServiceCredential } from 'web-api-client';
+
+import { TestEnvironment } from './common-types';
+import { functionalTestGroupTypes, TestGroupName } from './functional-test-group-types';
+import { TestRunner } from './runner/test-runner';
+import { TestContextData } from './test-group-data';
+import { FunctionalTestGroup } from './test-groups/functional-test-group';
+
+// tslint:disable: mocha-no-side-effect-code no-any no-unsafe-any mocha-unneeded-done strict-boolean-expressions
+
+describe('functional tests', () => {
+    // need to provide by user. DO NOT CHECK IN
+    const clientId = process.env.SP_CLIENT_ID || '';
+    const clientSecret = process.env.SP_PASSWORD || '';
+    const tenantId = process.env.SP_TENANT || '';
+    const apimName = process.env.APIM_Name || '';
+    const cosmosKey = process.env.COSMOS_DB_KEY || '';
+    const cosmosUrl = process.env.COSMOS_DB_URL || '';
+    // static values
+    const dbName = 'onDemandScanner';
+    const collectionName = 'scanRuns';
+    const releaseIdStub = 'release-id';
+    const runIdStub = 'run-id';
```

Comment:
Better to have real values here since it goes to app insight and will have to debug

Reply:
updated.